static.config.js

```diff
@@ -716,24 +716,30 @@ const makeAllPages = async (langCode, incrementalPageId) => {
     makeAllPages returns react-static data object with homepage
     and all built pages as children for '/en', '/es' and '/'
   */
-  const path = `/${!!langCode ? langCode : ''}`;
+  const path = `/${langCode || ''}`
   console.log(`- Building routes for ${path}...`);
 
   const client = createGraphQLClientsByLang(langCode);
 
   let pages = [];
   let after = '';
+  const batchSize = Number(process.env.REACT_STATIC_BATCH_SIZE) || 25
   while (true) {
-    const siteStructure = await client.request(allPagesQuery, { after: after });
+    const siteStructure = await client.request(allPagesQuery, { after: after, batchSize: batchSize});
     pages = pages.concat(siteStructure.allPages.edges);
     after = siteStructure.allPages.pageInfo.endCursor;
     if (!siteStructure.allPages.pageInfo.hasNextPage) {
       break;
     }
   }
 
-  // Build search index here before pages is altered.
-  const searchIndex = searchIndexBuilder(pages);
+  /**
+    Build search index here before pages are altered.
+    Hardcoding "withElasticsearch" and "indexName" for now.
```

Comment:
hardcoded for now, how will it get toggled in the future?

Reply:
I started writing a response and then I realized that it should just be written in an issue. Then I realized it should actually be written in an epic. https://github.com/cityofaustin/techstack/issues/4919